### ISSUE TITLE
Submodule peering example 

### ISSUE THREAD

**Issue by philipwigg** [terraform-google-modules/terraform-google-network#225]
This [Simple VPC Network Peering example](https://github.com/terraform-google-modules/terraform-google-network/tree/master/examples/submodule_network_peering) claims that ```This example creates a VPC Network peering between two VPCs.```.

To me, it seems to create three VPCs and create a `A->B` and `A->C` type peering setup.

Which should it be?

**Comment by bharathkkb** [comment]
Hi @philipwigg 
It creates `local-network <--> peer-network-1` and `local-network <--> peer-network-2`. For any two VPCs, A and B, the module `network-peering` creates  `A<-->B`

**Comment by philipwigg** [comment]
Thanks for your response. My point is, the [example](https://github.com/terraform-google-modules/terraform-google-network/tree/master/examples/submodule_network_peering) does not in fact create "a VPC peering between two VPCs" but rather it's 2 VPC peerings and three VPCs as you've referenced in your reply.

**Comment by bharathkkb** [comment]
@philipwigg yes happy to review PR to flesh out the docs. Also technically its 4 VPC peerings, each instance of the module creates 2 peerings A-to-B and B-to-A.
